### ISSUE TITLE
OnBlur doesn't seem to work with dropdown material ui

### ISSUE THREAD

**Issue by mr795137** [gokulakannant/react-form-input-validation#28]
everytime TextField from material ui is out of focus, the form refreshes itself (the error goes away), but when the select attribute is used on the TextField, the error doesn't go away even when i'm out of focus

Codesandbox : https://codesandbox.io/s/react-form-input-validation-demp-forked-ye1m1

**Comment by gokulakannant** [comment]
@mr795137 , I went through the codesandbox link which you have shared. I found some wrong implementation of the Material UI dropdown. I have updated the Material UI select in below codesandbox link

https://codesandbox.io/s/react-form-input-validation-demp-forked-0lxpx?file=/src/Form.js

Also, There is no onBlur event is fired by Material UI Select. Refer the below select dropdown API link, there is no such prop for Material UI select.

https://material-ui.com/api/select/
